api/apps/geoprocessing/src/marxan-sandboxed-runner/ports/input-files.ts

```diff
@@ -1,6 +1,19 @@
+export interface Settings {
+  PUNAME: string;
+  SPECNAME: string;
+  PUVSPRNAME: string;
+  BOUNDNAME: string;
+}
+
+export type Assets = {
+  // asset: keyof Settings;
+  url: string;
+  relativeDestination: string; // this
```

Comment:
I don't understand the comments 😅

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/293/commits/1d75fdcb196091abd991263952816edf49b0d7f2